src/commands/codepush/deployment/list.ts

```diff
@@ -17,6 +17,18 @@ import { scriptName } from "../../../util/misc";
 import { promiseMap } from "../../../util/misc/promise-map";
 import { formatDate } from "./lib/date-helper";
 
+const skipDeploymentMetrics = <T>() => {
```

Comment:
To be honest, I don't think it's worth using a decorator if it is applied to one function only. It makes code harder to read, but doesn't make it scalable in this case.

Reply:
Agreed so removed the decorator